Dockerfile

```diff
@@ -14,14 +15,15 @@ RUN apt-get update \
         nfs-common \
         unzip \
     && rm -rf /var/lib/apt/lists/* \
-    && pecl install memcached \
+    # Memcached 2.2.0 is the latest for PHP < 7
+    # see https://pecl.php.net/package/memcached
+    && pecl install memcached-2.2.0 \
```

Comment:
Choosing _not_ to bump the base image up to PHP 7, so I have to be more specific here